android/app/src/main/java/org/fossasia/openevent/activities/SearchActivity.java

```diff
@@ -134,103 +123,37 @@ public void handleIntent(Intent intent) {
         searchQuery(query);
     }
 
-    private void searchQuery(String constraint) {
-        results.clear();
-
-        if (!TextUtils.isEmpty(constraint)) {
-            String query = constraint.toLowerCase(Locale.getDefault());
-            String wildcardQuery = String.format("*%s*", query);
-            addResultsFromTracks(wildcardQuery);
-            addResultFromSessions(wildcardQuery);
-            addResultsFromSpeakers(wildcardQuery);
-            addResultsFromLocations(wildcardQuery);
+    @Override
+    public void onSaveInstanceState(Bundle bundle) {
+        if(searchViewModel != null){
+            searchView.setQuery(searchText, true);
```

Comment:
Can you explain what you're trying to do here ?

Reply:
@srv-twry As I dont have a setSearchText method in viewmodel to save the text there after configurations change I set the text in searchView to searchText

Reply:
Well , you could have created the method in case you needed them. Although here in my opinion you don't need to save it anywhere. Everytime you change the searchText in the search bar, the ViewModel is queried with that query and the searchText is saved in the ViewModel. So that means, the ViewModel always has the most updated searchText saved. No need to set it again.